src/main/java/step2/model/Lotteries.java

```diff
@@ -36,30 +39,32 @@ public static Lotteries of(int totalTicket, LottoFactory lottoFactory) {
 	}
 
 	public HashMap<Rank, Integer> isMatch(List<Integer> winNumber, int bonusNumber) {
-		winningLotto = WinningLotto.of(winNumber, bonusNumber);
+		WinningLotto winningLotto = WinningLotto.of(winNumber, bonusNumber);
 
 		for (Lotto lotto : lotteries) {
 			int countOfMatch = winningLotto.countOfMatch(lotto.getLotto());
-			Rank rank = Rank.of(countOfMatch);
-			countOfMatchBonus(countOfMatch, lotto.getLotto());
+			Rank rank = Rank.of(countOfMatch, winningLotto.isBonus(lotto.getLotto()));
```

Comment:
`Lotto`에서 getter인 `getLotto()`를 호출하는 대신 메시지를 보내서 처리하는 방식으로 구현해보면 어떨까요?
```suggestion
			int countOfMatch = winningLotto.countOfMatch(lotto);
			Rank rank = Rank.of(countOfMatch, winningLotto.isBonus(lotto));
```